src/doc/python_scripting/functions.rst

```diff
@@ -178,11 +178,15 @@ variableName : string
 
 inheritSIL : integer
     An integer flag indicating whether the plot should inherit the
-    active plot's SIL restriction.
+    active plot's SIL restriction. A zero value indicates that the SIL
```

Comment:
Yes, it does. Thats because the content here in this `.rst` file is also used as response to live `help(foo)` in CLI and the generation process is not sophisticated enough to, for example, take sentence per line formatting in `.rst` and re-wrap it for nicely formatted output in live CLI.

Reply:
So, how you format it here in the `.rst` file is going to be displayed almost verbatim in live CLI. If you create really long lines, those will wrap the terminal in CLI and look wierd. If you create line per sent, those will appear verbatim and, again, look wierd.